atomic_reactor/schemas/config.json

```diff
@@ -734,6 +734,18 @@
         "additionalProperties": false,
         "required": ["name", "value"]
       }
+    },
+    "image_size_limitation": {
+      "description": "Define maximum image size for regular image checked before push",
+      "type": "object",
+      "properties": {
+        "regular_image": {
+          "description": "Maximum size of regular images. Set to 0 or omit to skip the check",
```

Comment:
IMO 0 to skip check is kinda confusing, omit to skip should be good enough. If someone sets 0, build should probably just fail when checking size.

Reply:
In configs I'm using, 0 usually means unlimited.

Reply:
How does OSBS handle such value in a consistent way? On the other hand, IMO, if 0 makes no sense for this case, it could be easy to set minimum value to 1 in this schema to validate it.

Reply:
Not sure about OSBS, but in python, unlimited is usually indicated with `-1`. I don't really mind though, if 0 for unlimited is common then we can keep it this way.

Setting minimum to 1 probably does not make that much sense, in practice both 1 and 0 would make all builds fail anyway.

Reply:
IMO, 0 does make sense for meaning unlimited here.